tests/docs/field-initializer-context-reference.test.sh

```
@@ -30,9 +30,17 @@ CHEATSHEET="skills/unity-vrc-udon-sharp/CHEATSHEET.md"
 
 assert_contains "$SKILL" "Editor-evaluated field initializers"
 assert_contains "$SKILL" "Udon runtime code"
+assert_contains "$SKILL" "Random.Range"
+assert_contains "$SKILL" "baked default"
+assert_contains "$SKILL" "per-instance, per-client, or per-session randomness"
+assert_contains "$SKILL" "lazy-init guard"
 assert_contains "$RULES" "## Blocked in Udon Runtime"
 assert_contains "$RULES" "Editor-Evaluated Field Initializers"
+assert_contains "$RULES" 'Nondeterministic calls such as `Random.Range`'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)